nasa-client/asteroid-client.js

```diff
@@ -16,6 +16,19 @@ asteroidApi.interceptors.request.use((config) => {
     return Promise.reject(error);
 });
 
+function convertToResponse(asteroids) {
+    return Object.values(asteroids.near_earth_objects).flatMap(date => {
+        return date.map(asteroid => ({
```

Comment:
u can use destructurization props here. immediately retrieve the parameters to be used from the object